cmd/drivers/hyperkit/main.go

```diff
@@ -19,10 +19,18 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/docker/machine/libmachine/drivers/plugin"
 	"k8s.io/minikube/pkg/drivers/hyperkit"
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "--version" {
```

Comment:
please change this to "version" instead of "--version" in both here and kvm2 driver, to be consistent with minikube. it took me a couple miniutes to figure out this one is different than minikube